src/StructTypes.jl

```diff
@@ -656,7 +673,7 @@ mappings will be applied, and the function will be passed the Julia field name.
     f_applied = false
     # unroll the first 32 field checks to avoid dynamic dispatch if possible
     Base.@nif(
-        32,
+        33,
```

Comment:
Why is this changed?

Reply:
Ok, saw your comment in the PR

Reply:
Yeah, without this change, the 32nd field of a struct will never get applied.